frontend/public/components/routes/create-route.tsx

```diff
@@ -196,11 +223,51 @@ export class CreateRoute extends React.Component<null, CreateRouteState> {
       }));
   }
 
+  addAltServiceEntry(){
```

Comment:
```suggestion
  addAltServiceEntry = () => {
```

Then you can use

```
onClick={this.addAltServiceEntry}
```

below